### PR TITLE
Feature/rdmpdev

### PR DESCRIPTION
**Pull request by jas88**
Build directly against RDMP repo, instead of pulling binary releases from nuget.org (avoids delay in updating plugin for new RDMP releases, also enables better pre-release testing).

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -25,6 +25,11 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - uses: actions/checkout@v3
```
Comment (rdteviotdale) [inline .github/workflows/codeql.yml:28]: Duplication of "uses", typo?

File: .github/workflows/dotnet-core.yml [hunk 2]
```diff
@@ -12,68 +12,88 @@ jobs:
     runs-on: windows-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/checkout@v3
```
Comment (rdteviotdale) [inline .github/workflows/dotnet-core.yml:15]: Duplication of "uses", typo?

**Review by rdteviotdale — CHANGES_REQUESTED** [review]

File: .github/workflows/dotnet-core.yml (hunk 2 above)
Comment (jas88) [reply .github/workflows/dotnet-core.yml:15]: Not typo - it's invoking the Checkout action twice, once for the main repo and again to get a copy of the RDMP source to build against.

File: .github/workflows/codeql.yml (hunk 1 above)
Comment (jas88) [reply .github/workflows/codeql.yml:28]: Not typo - it's invoking the Checkout action twice, once for the main repo and again to get a copy of the RDMP source to build against.

**Review by rdteviotdale — APPROVED** [review]

**Pull request merged**